src/optionfunctions.js

```diff
@@ -103,4 +103,8 @@ export const showFurigana = function(show) {
 
 export const showEmojis = function(show) {
     document.getElementById("conjugation-inquery-text").className = show ? "" : "hide-emojis";
+}
+
+export const showStreak = function(show) {
+    document.getElementById("streak-container").className = show ? "" : "hide-streak";
```

Comment:
I want to keep some padding between the "verb-container" and the top of the "top-container" when the user hides the streaks. I think the simplest way is to keep the streak-container which has padding but hide the individual streaks like this:
```suggestion
    document.querySelectorAll(".streak").forEach(s => {
        if (show) {
            s.classList.remove("display-none")
        } else {
            s.classList.add("display-none")
        }
    });
```